include/ck/tensor_operation/gpu/device/device_gemm_reduce_xdl_cshuffle_v2.hpp

```diff
@@ -0,0 +1,728 @@
+#pragma once
```

Comment:
Please update existing ```DeviceGemmReduce_Xdl_CShuffle``` instead of adding a new one

Reply:
b9203997a4b364652fe6fd8aec1a99b00a38ddc6